client/src/components/Task/Task.js

```diff
@@ -67,38 +61,51 @@ export default function Task(props) {
     React.useEffect(() => {
         if (dragging) {
             setOptionsVisible(false);
-            setEditingContent(false);
         }
-    }, [dragging])
+    }, [dragging]);
+
+    //Effect when main store content is changed
+    React.useEffect(() => {
+        setTemporaryContent({content: content.content, lastEdit: new Date().getSeconds()});
```

Comment:
Just curious, when will mainContent.lastEdit > temporaryContent.lastEdit

correct me if im wrong, so even if we update the temporaryContent instantly after the mainContent is updated, there will still be a short period of time whereby the temporaryContent.lastEdit will be < mainContent.lastEdit?

Reply:
mainContent > temporaryContent for the single rerender immediately after dragEnd happens.

There's no way for us to "immediately" update temporaryContent with the current structure, the earliest we can do is 1 frame after mainContent was updated. So without lastEdit, we will have 1 wrong frame.